testing/simapp/app.go

```diff
@@ -870,4 +871,9 @@ func (app *SimApp) setupUpgradeHandlers() {
 		simappupgrades.DefaultUpgradeName,
 		simappupgrades.CreateDefaultUpgradeHandler(app.mm, app.configurator),
 	)
+
+	app.UpgradeKeeper.SetUpgradeHandler(
+		v6.UpgradeName,
+		v6.CreateUpgradeHandler(app.mm, app.configurator, app.appCodec, app.keys[capabilitytypes.ModuleName], app.CapabilityKeeper, ibcmock.ModuleName+icacontrollertypes.SubModuleName),
```

Comment:
It'd probably make sense to add a comment here on what the value of the "moduleName" should be as this will vary from chain to chain

Reply:
Updated with some commented/doc info